src/main/resources/application-local.yaml

```diff
@@ -1,3 +1,6 @@
+server:
+  port: 8081
```

Comment:
does it have any purpose or u just forgot to delete this?

Reply:
removed